### PR TITLE
DSDEGP-2017 Cancel button disabled

### PR DESCRIPTION
**Pull request by tadeork**
### Description

_After Cancel downloads or uploads, the button Cancel is not disabled._
_Explain the **motivation** for making this change. What existing problem does the pull request solve?_
_Mention any JIRA issues fixed, addressed or otherwise related to this pull request, including issue numbers or hard links for issues in other repos._

----

### Checklist

- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: Ensure that you have added or updated tests
- [ ] **Submitter**: If you're adding/switching libraries or otherwise changing the design, update the [design documentation](https://broadinstitute.atlassian.net/wiki/pages/viewpage.action?pageId=189431877).
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * Add notes on what you've tested
* Review cycle:
  * Team may comment on PR at will
  * **Reviewer assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to reviewer** for further feedback
- [ ] **Submitter**: Verify all tests go green, including CI tests
- [ ] **Submitter**: Squash commits and merge to develop
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Move the JIRA issue to QA once this checklist is completed

### CODE REVIEW

File: src/app/file-manager/transferables-grid/transferables-grid.component.ts [hunk 1]
```diff
@@ -156,6 +166,7 @@ export class TransferablesGridComponent implements OnInit, AfterViewInit {
 
   startUpload(files: Item[]) {
     this.limitTransferables.controlLimitItems(files, Type.UPLOAD, ItemStatus.UPLOADING);
+    this.uploadInProgress = true;
```
Comment (pshapiro4broad) [inline src/app/file-manager/transferables-grid/transferables-grid.component.ts]: Why is this needed here? Do we also need this in `startDownload()`?

Comment (tadeork) [reply src/app/file-manager/transferables-grid/transferables-grid.component.ts]: No, actually we no need it here either.

**Review by pshapiro4broad — APPROVED** [review]
